webapp/channels/src/components/admin_console/remove_file_setting.tsx

```diff
@@ -16,6 +16,7 @@ type Props = SettingsProps & {
     fileName: string;
     onSubmit: (arg0: string, arg1: () => void) => void;
     disabled?: boolean;
+    onSetData?: any;
```

Comment:
This should be a function of some kind.

Reply:
removing it because it is not being used anywhere